src/gei/FileDownloader.cs

```diff
@@ -0,0 +1,26 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter;
+
+public class FileDownloader
+{
+    private readonly HttpClient _client;
+    private readonly OctoLogger _octoLogger;
+
+    public FileDownloader(HttpClient client, OctoLogger octoLogger)
+    {
+        _client = client;
+        _octoLogger = octoLogger;
+    }
+
+    public virtual async Task<byte[]> DownloadArchive(string fromUrl)
```

Comment:
Since this is a generic file downloader I would call this method just `Download`.